src/common/card/card.tsx

```diff
@@ -1,14 +1,14 @@
 import {FC} from "react";
 import {Link, useNavigate} from "react-router-dom";
-import {Post, User} from "@/services/api/gen";
+import {Post} from "@/services/api/gen";
 
 type UserActivity = {
-  user: User | undefined;
+  isSelf: boolean;
```

Comment:
shouldn't take this prop, as we can get it inside the component,
a component should take the fewer props possible for simplicity